ripple/src/main/webapp/script.js

```diff
@@ -355,6 +355,30 @@ function isWithinRange(target, startRange, endRange, lessThanAndEquals=false) {
   return false;
 }
 
+/* Function that creates a generic carousel image element. Used in businessdetails and managebusiness. */
+function createCarouselElement(blobKey, id) {
+  if (blobKey != null) {
+    console.log(blobKey);
+    var content = `
+        <div class="carousel-item col-md-4">
+        <div class="card">
+            <img class="card-img-top img-fluid" id="card-dynamic-image" src="/serve?blob-key=${blobKey}"></img>
+        </div>
+        </div>
+    `;
+    console.log(content);
+    // Append newly created card element to the container
+    var carouselElement = document.getElementById(id);
+    var carouselInner = document.getElementById("firstRow");
+    carouselInner.innerHTML = carouselInner.innerHTML + content + "\n";  
+    // Only want to append the string active once
+    if (!carouselInner.firstElementChild.className.includes("active")) {
+        carouselInner.firstElementChild.className += " active";
+    }
+    $(carouselElement).carousel({slide : true, interval : false});
+    }
```

Comment:
should line up with the opening bracket of the if block

Reply:
fixed